static/js/ui_init.js

```diff
@@ -203,7 +203,9 @@ function initialize_compose_box() {
             embedded: $("#compose").attr("data-embedded") === "",
             file_upload_enabled: page_params.max_file_upload_size_mib > 0,
             giphy_enabled: giphy.is_giphy_enabled(),
-            scroll_to_bottom_key: common.has_mac_keyboard() ? "Fn + Right arrow" : "End",
+            scroll_to_bottom_key: common.has_mac_keyboard()
+                ? "Fn + <i class='tooltip_right_arrow fa fa-arrow-right'></i>"
```

Comment:
You don’t need Font Awesome to write a → character.

Reply:
Updated.
![DeepinScreenshot_select-area_20220224131207](https://user-images.githubusercontent.com/25124304/155480118-4b52c2ae-d3a1-4e14-99ea-ca766d7284c2.png)